### DISCUSSION TITLE
[@types/vue-router] Incorrect package name in deprecation message

### DISCUSSION THREAD

**Discussion by luc122c** [DefinitelyTyped/DefinitelyTyped#63489] [Issues with a @types package]
> This is a stub types definition for vue-router (https://github.com/vuejs/vue-router). ecmarkup provides its own type definitions, so you don't need @types/ecmarkup installed!

The author message on [this package](https://www.npmjs.com/package/@types/vue-router) should say `@types/vue-router`

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
the message is autogenerated

**Reply by luc122c** [reply]
Then it seems be autogenerated incorrectly.

**Reply by martin-badin** [reply]
you are right but I don't think it is possible to change it.

**Reply by luc122c** [reply]
Surely the code to autogenerate the message is in this repo somewhere? Or is it done on NPM's side?
